include/wx/textctrl.h

```diff
@@ -321,6 +329,13 @@ class WXDLLIMPEXP_CORE wxTextAttr
     void SetFontWeight(wxFontWeight fontWeight) { m_fontWeight = fontWeight; m_flags |= wxTEXT_ATTR_FONT_WEIGHT; }
     void SetFontFaceName(const wxString& faceName) { m_fontFaceName = faceName; m_flags |= wxTEXT_ATTR_FONT_FACE; }
     void SetFontUnderlined(bool underlined) { m_fontUnderlined = underlined; m_flags |= wxTEXT_ATTR_FONT_UNDERLINE; }
+    void SetFontUnderline(bool underlined, wxTextAttrUnderlineType type = wxTEXT_ATTR_UNDERLINE_SOLID, wxColour colour = *wxBLACK)
```

Comment:
@MaartenBent ,

You mean make a case of the more generic functionality I'm putting in?

I guess I can do that...

Reply:
`SetFontUnderline` with default arguments does (or should do, see colour remark) the same as `SetFontUnderlined`. There is no need to keep both.

Because someone probably is already using `SetFontUnderlined` in their code, I think it is better to rename your new function to `SetFontUnderlined` so their code will not break.